src/Mod/Fem/feminout/importZ88O2Results.py

```diff
@@ -68,8 +68,8 @@ def insert(
 # ********* module specific methods *********
 def import_z88_disp(
     filename,
-    analysis = None,
-    result_name_prefix = None
+    analysis=None,
```

Comment:
Why this change? usually we use spaces around binary operators.

Reply:
yes,  but not inside  the method parameter